iglo/league/models.py

```diff
@@ -282,18 +284,20 @@ def _assign_players_to_groups(self, players: list["Player"]) -> None:
         last_group = None
         for group_order in range(math.ceil(len(players) / self.players_per_group)):
             group_players = players[group_order * self.players_per_group : (group_order + 1) * self.players_per_group]
+            # Note: is_egd is deprecated, we now check per-game eligibility
+            # but we keep this for backward compatibility
             is_egd = all(p.egd_approval for p in group_players)
             if len(group_players) < max((self.players_per_group - 1), 2) and last_group:
                 group = last_group
                 group.type = GroupType.MCMAHON
-                group.is_egd = group.is_egd and is_egd
+                group.is_egd = group.is_egd and is_egd  # Deprecated - games are now eligible based on individual player approvals
```

Comment:
we don't need to repeat the deprecation notice everywhere it should be done in the entity class. (group.is_egd should be deprecated)

Reply:
Done.

Reply:
the comments are still there